ons_alpha/jinja2/templates/pages/manage_cookie_settings.html

```diff
@@ -0,0 +1,388 @@
+{% extends "templates/base.html" %}
+{% from "components/radios/_macro.njk" import onsRadios %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/details/_macro.njk" import onsDetails %}
+{% from "components/table/_macro.njk" import onsTable %}
+
+{% block main %}
+    <div class="ons-page__body">
```

Comment:
NOTE: This content is an ever-so-slightly adapted version of the pattern provided by https://service-manual.ons.gov.uk/design-system/patterns/cookies-settings